frontend/src/metabase/query_builder/components/notebook/steps/BreakoutStep/BreakoutStep.unit.spec.tsx

```diff
@@ -35,14 +35,14 @@ function createQueryWithBinning(bucketName = "10 bins") {
   return { query, columnInfo: Lib.displayInfo(query, 0, columnWithBinning) };
 }
 
-function createQueryWithTemporalBreakout() {
+function createQueryWithTemporalBreakout(bucketName = "Quarter") {
```

Comment:
IMO `bucketName` should not have default value

Reply:
removed